src/app/portlets/dot-edit-page/content/dot-edit-content.component.spec.ts

```diff
@@ -779,13 +779,25 @@ describe('DotEditContentComponent', () => {
 
             spyOn(dotEditPageService, 'save').and.returnValue(Observable.of(true));
             spyOn(dotEditContentHtmlService, 'getContentModel').and.returnValue({});
+            spyOn(dotEditContentHtmlService, 'setContaintersSameHeight');
+
             fixture.detectChanges();
 
             dotEditContentHtmlService.pageModel$.next(model);
             fixture.detectChanges();
 
             dotEditContentHtmlService.pageModel$.next(newModel);
             expect(dotEditPageService.save).toHaveBeenCalledTimes(2);
+            expect(dotEditContentHtmlService.setContaintersSameHeight).toHaveBeenCalled();
         });
     });
+
+    it('should set listener to change containers height', () => {
+        spyOn(dotEditContentHtmlService, 'setContaintersChangeHeightListener');
+        fixture.detectChanges();
+        component.pageState.state.mode = 0;
+        component.onLoad(Event);
+        expect(dotEditContentHtmlService.setContaintersChangeHeightListener).toHaveBeenCalled();
+    });
+
```

Comment:
I think we miss test where this condition is false

https://github.com/dotCMS/core-web/pull/668/files#diff-c05bee8f4c4da08b8a76f1454e106074R159

and dotEditContentHtmlService.setContaintersChangeHeightListener is not call